include/boost/geometry/formulas/spherical.hpp

```diff
@@ -217,6 +219,47 @@ inline int azimuth_side_value(T const& azi_a1_p, T const& azi_a1_a2)
         : 1; // left
 }
 
+template <typename CT, typename Sphere>
+inline result_direct<CT> spherical_direct(CT const& lon1,
```

Comment:
An alternative formula would convert to 3D cartesian ECEF and calculate the result using vector algebra. This is how it's done in the spherical intersection and side strategies. So should be careful to not mix things. But AFAIU this formula is a building block for other geographic formulas and not used in spherical for now. Is my understanding correct?

Reply:
Yes this is not used anywhere for now, I added it for completeness. I could be used in geographic pt-seg distance formula for optimizations.